yarn-project/bot/src/bot.ts

```diff
@@ -68,7 +68,14 @@ export class Bot {
     this.log.verbose(`Sending tx`, logCtx);
     const tx = batch.send(opts);
 
-    this.log.verbose(`Awaiting tx ${tx.getTxHash()} to be mined (timeout ${this.config.txMinedWaitSeconds}s)`, logCtx);
+    const txHash = tx.getTxHash();
+
+    if (this.config.txMinedWaitSeconds === 0) {
+      this.log.verbose(`Transaction ${txHash} sent, not waiting for it to be mined`);
```

Comment:
```suggestion
      this.log.info(`Transaction ${txHash} sent, not waiting for it to be mined`);
```
So we always hit at least one `info`-level log per tx sent